src/components/login/login.scss

```diff
@@ -1,20 +1,20 @@
-.login-button {
-  width: 100%;
-}
-.ah-modal-content {
-  margin: 0 auto;
-  max-height: calc(100vh - 40px);
-  width: 500px;
-  padding: 50px;
-  background-color: white;
-}
-.modal-background {
-  background-color: rgba(10, 10, 10, 0.75);
-}
-.login-heading {
-  font-family: sans-serif;
-  color: #35438f;
-  text-align: center;
-  padding-bottom: 20px;
-  padding-top: 20px;
-}
+//.login-button {
```

Comment:
Nice Job. Please remove the commented code if it is no longer needed.

Reply:
I will do that now, thanks